build.ps1

```diff
@@ -159,7 +159,7 @@ function RunTests {
 }
 
 function UnitTests {
-    Invoke-Execute { RunTests -Filter "*.Tests" }
+    Invoke-Execute { RunTests -Filter "*.Api.Tests" }
```

Comment:
We will have other unit test projects in the future. Perhaps now is a good time to set a better naming convention.

What if we use `xyz.Tests.<type>`? For example, `EdFi.DataManagement.Api.Tests.Unit` and `EdFi.DataManagement.Api.Tests.E2E`?

Reply:
I will make the change